packages/babel-preset-theme-ui/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "babel-preset-theme-ui",
+  "version": "0.2.49",
+  "description": "Babel preset to enable the theme-ui custom pragma by default.",
+  "keywords": [
+    "babel",
+    "babel-preset",
+    "theme-ui"
+  ],
+  "author": "Rich Lewis <opensource@richlew.is>",
+  "license": "MIT",
+  "main": "src/index.js",
+  "repository": "system-ui/theme-ui",
+  "dependencies": {
+    "@babel/plugin-transform-react-jsx": "^7.7.4",
+    "@wordpress/babel-plugin-import-jsx-pragma": "^2.3.0"
```

Comment:
Why not just use `@emotion/babel-plugin-jsx-pragmatic` since `@emotion/babel-preset-css-prop` [does](https://github.com/emotion-js/emotion/blob/master/packages/babel-preset-css-prop/src/index.js#L19-L24) exactly that?

Reply:
I hadn't seen the preset (haven't used emotion much), the wp one came up first in my quick google search. Look like they do the same thing, but the emotion one seems much more sensible to use. I can update.